katello/hooks/pre/29-install_postgresql.rb

```diff
@@ -0,0 +1,10 @@
+# Working around https://tickets.puppetlabs.com/browse/PUP-2169
+if local_postgresql? && facts[:os][:release][:major] == '7' && !app_value(:upgrade)
```

Comment:
For now, I don't want this messing with the upgrade workflow given thats a very orchestrated change.